### ISSUE TITLE
Researching bronze working breaks game

### ISSUE THREAD

**Issue by Cindela** [Thalassicus/cep-bnw#262]
so the main problem is that when i research bronze working i can no longer chose production for my cities, and it doesn't let me end my turn until I chose new production. also a side problem that might be related, i can not chose any research from the tech tree past the ancient era and also i can't chose bronze working from the tech tree.

**Comment by Thalassicus** [comment]
Could you attach your log file after you encounter the problem? Instructions are here:

http://forums.civfanatics.com/showthread.php?p=10221327#post10221327

**Comment by Cindela** [comment]
![lua log](https://cloud.githubusercontent.com/assets/7281527/2691108/cf15a248-c373-11e3-8599-85727d73e4b7.png)
idk if i did that right. but i followed the instructions up until the part where it said to end my turn, i could not end the turn due to being unable to chose production

**Comment by GrantSP** [comment]
Your log shows you are using another mod that adds in new resources, are those new resources becoming available at Bronze Working?
Perhaps you could provide a link to the mod in question?

As a test try a game without that mod and see if you can progress okay.

**Comment by Cindela** [comment]
bronze working is blank as is pottery. and your idea about the resources mod didn't work. took it out and it still broke at bronze. this only started once i began using the communitas Ai and tools.

**Comment by GrantSP** [comment]
There is another mod conflicting with CEP.
I have just now run a test game with **only** CEP and there is no problem once _Bronze Working_ is researched.

Do you have, and know how to use, the FireTuner tool?
I believe there is a information panel that becomes active, in the background, at _Bronze Working_ and it is not being cleared.
This is not allowing the TechTree or ProductionPopup panels to become available.
One of the other mods you are running is firing this panel and it isn't allowing the game to progress until it is cleared.

**Comment by Cindela** [comment]
hmm i got rid of a windmill mod and it let me get a few turns past bronze working. im gona get rid of the other builing mod i have.

**Comment by Cindela** [comment]
and i dont know what the firetuner tool is.

**Comment by Cindela** [comment]
nope the building mods aren't the problem.

**Comment by Cindela** [comment]
weird i just noticed that when i research bronze it doesn't change the finished mining to bronze and turns

**Comment by ghost** [comment]
I would try deleting all mods and start over following these steps - http://forums.civfanatics.com/showthread.php?p=10901714#post10901714

I have had no issues like the one you have reported with the latest version and I have played a number of games.  I wonder if something is stuck in the cache.

**Comment by Cindela** [comment]
ok i tried something expecting the communitas mods to not be able to work. i tried removing the BNW dlc and suddenly bronze working had things in it again. well i guess the communitas mods are a replacement for BNW, not an upgrade to it.

**Comment by Thalassicus** [comment]
No, the communitas mods require BNW. Have you tried the things on the [common checklist for playing mods](http://forums.civfanatics.com/showthread.php?p=10901714#post10901714) jbmoll linked? The main things you want to do are:
1. Remove all mods except the 4 communitas mods (map, ai & tools, enhanced gameplay, interface)
2. Delete the Cache folder.
3. Start a game.

Something else you might try is verifying the Civ V game cache through steam.

**Comment by Cindela** [comment]
would removing all mods except communitas from the active list work or do i actually have to delete them?

**Comment by Thalassicus** [comment]
I recommend moving the other mods to a backup folder outside the mods folder. If it works with just the 4 communitas mods installed, you can add & enable 1 other mod back in to your mods folder at a time, until you find the one that breaks things.

**Comment by Cindela** [comment]
ok thanks

**Comment by Cindela** [comment]
god frigin seriously. so the problem is a new civilization i really like playing as. the eugenicists. some how it messes with the research tree.
